dev-python/asciinema/python3_asciinema-1.4.0.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="Command line recorder for asciinema.org service"
+DESCRIPTION="asciinema lets you easily record terminal sessions and replay \
+them in a terminal as well as in a web browser."
+HOMEPAGE="https://asciinema.org/"
+COPYRIGHT="2011-2017 Marcin Kulik"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/asciinema/asciinema/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="841a55b0f51988d5e155e99badbd6ce5cf3b43cca2ba15cd20c971a19719dc9a"
+SOURCE_DIR="asciinema-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	python3_asciinema$secondaryArchSuffix = $portVersion
+	cmd:asciinema$secondaryArchSuffix
+"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:python3
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	setuptools_python3
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:python3
+"
+
+BUILD()
+{
+	$portPackageLinksDir/cmd~python3/bin/python3 setup.py build
+}
+
+INSTALL()
+{
+	# GENERIC: all python_setuptools-based installs need this
+	pythonVersion=$(python3 --version 2>&1 | sed 's/Python //' | head -c3)
+	installLocation=$prefix/lib/python$pythonVersion/vendor-packages/
+	export PYTHONPATH=$installLocation:$PYTHONPATH
+	mkdir -p $installLocation
+
+	$portPackageLinksDir/cmd~python3/bin/python3 setup.py install \
+		--prefix=$prefix --skip-build
```

Comment:
Remove --skip-build
If files aren't correctly installed in 
then you could try to add --root=/

Reply:
you left --skip-build